README.md

```diff
@@ -88,23 +82,26 @@ In order to run the highest resolution supported by the encoder, at least 64GB o
 
 * __Build Requirements__
 	-	Visual Studio* 2017 (can be downloaded [here](https://www.visualstudio.com/vs/older-downloads/))
-	-	YASM Assembler version 1.2.0 or later
-	-	Download the yasm package from the following [link](http://www.tortall.net/projects/yasm/releases/vsyasm-1.2.0-win64.zip)
-	-	Unzip the package and in the vsyasm.props file and make the following change:
-		-	"(IntDir)"  into  "(IntDir)\%(Filename).obj"
-	-	Copy the Yasm files [all 4 of them] under the bin folder of your Visual Studio* installation folder ($(VCInstallDir)) e.g.: C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin (the path may differ based on the version of visual studio installed)
+	-	CMake 3.5 or later (can be downloaded [here](https://github.com/Kitware/CMake/releases/download/v3.13.0/cmake-3.13.0-win64-x64.msi))
+	-   YASM Assembler version 1.2.0 or later
+	-	Download the yasm exe from the following [link](http://www.tortall.net/projects/yasm/releases/yasm-1.3.0-win64.exe)
+	-	rename yasm-1.3.0-win64.exe to yasm.exe
+	-   Copy yasm.exe into a location that is in the PATH environment variable
 
 * __Build Instructions__
-	-	Open the “Build\VS\ebHevcEnc.sln” file and build the solution
+	-	Generate the Visual Studio* 2017 project files by following the steps below in a windows command line prompt:
+		-	cd Build\windows
+		-   build.bat
+	-   Open "svt-hevc.sln" using Visual Studio* 2017 and build the solution
 
 * __Binaries Location__
 	-	Post build, binaries can be found under Bin/Release and / or Bin/Debug
 
 * __Installation__
 -	For the binaries to operate properly on your system, the following conditions have to be met:
 	-	On any of the Windows* Operating Systems listed in the OS requirements section, install Visual Studio 2017
-	-	Once the installation is complete, copy the binaries to a location making sure that both the sample application “ebHevcEncApp.exe” and library “ebHevcEncLib.dll” are in the same folder.
-	-	Open the command prompt window at the chosen location and run the sample application to encode.
+	-	Once the installation is complete, copy the binaries to a location making sure that both the sample application "SvtHevcEncApp.exe” and library "SvtHevcEnc.dll” are in the same folder.
+	-	Open the command prompt window at the chosen location and run the sample application to encode. SvtHevcEncApp.exe -i [in.yuv] -w [width] -h [height] -fps [int_framerate] -b [out.265].
 	-	Sample application supports reading from pipe. E.g. ffmpeg -i [input.mp4] -nostdin -f rawvideo -pix_fmt yuv420p - | ebHevcEncApp.exe -i stdin -n [number_of_frames_to_encode] -w [width] -h [height].
```

Comment:
Is -fps now a required(not optional) parameter?

Reply:
removed -fps as it is optional